### PR TITLE
Add browser support to frontend docs

### PR DESCRIPTION
**Pull request by benhalpern**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Optimization
- [x] Documentation Update

## Description
Adds documentation about the browsers we support.

### CODE REVIEW

File: docs/frontend/browser-support.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Browser Support
+---
+
+# Browser Supprot
```
Comment (nickytonline) [inline docs/frontend/browser-support.md]:
```suggestion
# Browser Support
```

File: docs/frontend/browser-support.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+This application supports the following browsers
+
+- Chrome >= 70
```
Comment (nickytonline) [inline docs/frontend/browser-support.md]: If these are the official browsers we support, we should probably update the browser list to reflect that, https://github.com/thepracticaldev/dev.to/blob/master/.babelrc#L3

File: docs/frontend/browser-support.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Browser Support
+---
+
+# Browser Support
+
+This application supports the following browsers
```
Comment (rhymes) [inline docs/frontend/browser-support.md]: The list should also include mobile browsers.

**Comment by rhymes** [comment]
@nickytonline, I agree with you, the issue is that if we modify `.babelrc` with: `chrome>=70,firefox>=60,safari>=12,opera>=64,edge>=81` we lose mobile browsers.

You can check with: 

- now with `> %1`: https://browserl.ist/?q=%3E+1%25
- if we explicitly change it to https://browserl.ist/?q=chrome%3E%3D70%2Cfirefox%3E%3D60%2Csafari%3E%3D12%2Copera%3E%3D64%2Cedge%3E%3D81 we lose iOS Safari, the Samsung Browser and Opera Mini

**Comment by nickytonline** [comment]
True. It could become tedious to customize the browser list for that whole list of mobile browsers as well. A quick Google found an interesting tool, https://blog.jakoblind.no/babel-preset-env-custom-browserlistrc/. I’m not positive l, but I’m assuming that dev uses Google Analytics?

**Comment by mstruve** [comment]
Hey @benhalpern, since this has been inactive for a while I am going to close this for now. Feel free to reopen and revive if you want to get it pushed through!

**Pull request closed without merging**